dms-frontend/src/configs/auth.js

```diff
@@ -1,7 +1,7 @@
 export default {
-  meEndpoint: '/auth/me',
-  loginEndpoint: '/jwt/login',
-  registerEndpoint: '/jwt/register',
+  meEndpoint: '/profile',
```

Comment:
Its your personal choice again, why don't we just call them `/me`, `/login`, `/register`?

Reply:
Resolved!